src/main/java/se/sundsvall/byggrintegrator/integration/byggr/ByggrIntegrationMapper.java

```diff
@@ -110,9 +111,14 @@ private ByggrErrandDto toByggErrandDto(Arende arende) {
 
 	private String toPropertyDesignation(Arende arende) {
 		return ofNullable(arende)
-			.map(a -> (ArendeFastighet) a.getObjektLista().getAbstractArendeObjekt().getFirst())
+			.map(Arende::getObjektLista)
+			.map(ArrayOfAbstractArendeObjekt2::getAbstractArendeObjekt)
+			.filter(list -> !list.isEmpty())
```

Comment:
Can list be null here? If you use CollectionUtils::isNotEmpty, then you will be null safe as it checks if list object is null and if list object is not null, then if list object is empty. If you use this, the add a filter before this row with `.filter(Objects::nonNull)` or equal.